packages/amplify-cli/src/packageManagerHelpers.ts

```diff
@@ -24,6 +25,28 @@ export async function getPackageManager() {
   return undefined;
 }
 
+/**
+ * Determine the starting command of the current project
+ *
+ * @return {string} 'serve' or 'start'
+ */
+export async function getPackageManagerCommand() {
+  let scripts;
+  if (fs.existsSync(packageJsonDir)) {
+    const pjson = fs.readFileSync(packageJsonDir).toString();
+    const parsedPjson = JSON.parse(pjson);
+    scripts = parsedPjson['scripts'];
+  }
```

Comment:
use JSONUtilities from `amplify-cli-core` for this

Reply:
Fixed